### DISCUSSION TITLE
Section 05: Unable to load going modular notebooks

### DISCUSSION THREAD

**Discussion by monika0603** [mrdbourke/pytorch-deep-learning#1343] [Q&A]
[
<img width="1130" height="252" alt="Screenshot 2026-04-29 at 8 32 36 AM" src="https://github.com/user-attachments/assets/c67cec66-7dc8-4fe2-9844-142f078f02fb" />
](url)

I am unable to open the notebooks for going modular section. Please help. I have attached a screen shot of the error I get.

**Comment by monika0603** [comment]
never mind I was able to solve the problem. 

`import nbformat

path = "05_pytorch_going_modular_script_mode.ipynb"

nb = nbformat.read(path, as_version=4)

# Remove bad widgets metadata if present
if "widgets" in nb.metadata:
    del nb.metadata["widgets"]

nbformat.write(nb, path)`

**Comment by aasimansari1** [comment]
Hope you get it sorted! While you're learning PyTorch — I just open-sourced **[ML/AI Interview Prep](https://github.com/aasimansari1/ml-interview-prep)** with a complete PyTorch training loop template (grad clipping, early stopping, learning rate scheduling, evaluation), custom Dataset class, and 500+ ML interview Q&A. Might be useful alongside this course. ⭐ if helpful!
